guides/common/modules/proc_adding-openstack-details-to-a-compute-profile.adoc

```diff
@@ -22,6 +22,12 @@ If not selected, the instance boots the image directly.
 . Click *Submit* to save the compute profile.
 
 .CLI procedure
-
-The compute profile CLI commands are not yet implemented in {ProjectName}.
-As an alternative, you can include the same settings directly during the host creation process.
+. To add Openstack details to a compute profile, use the following command:
```

Comment:
Elsewhere we use the `{OpenStack}` attribute, which for Red Hat uses the brand name. Otherwise it also needs to the correct casing. So either
```suggestion
. To add OpenStack details to a compute profile, use the following command:
```

Or
```suggestion
. To add {OpenStack} details to a compute profile, use the following command:
```